index.html

```diff
@@ -318,41 +318,30 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li>Patronizing language or behavior:
+            <ul>
+              <li>Intentionally or unintentionally making assumptions about the skills or knowledge of others, such as using language that implies the audience is uninformed on a topic (e.g. interjections like "I can't believe you don't know about [topic]").
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled due to their characteristics (e.g., “So easy your grandmother could do it”).
```

Comment:
s/grandmother/anyone/

Reply:
I think I'd much rather explain why this particular example is problematic than try to change it. This is a pretty common colloquialism too, changing it would take some of the familiarity away.